packages/front-end/components/Metrics/MetricsList.tsx

```diff
@@ -73,9 +73,18 @@ export function useCombinedMetrics({
 
   const combinedMetrics = [
     ...inlineMetrics.map((m) => {
-      const canDuplicate = permissionsUtil.canCreateMetric(m);
-      const canEdit = permissionsUtil.canUpdateMetric(m, {});
-      const canDelete = permissionsUtil.canDeleteMetric(m);
+      const canDuplicate =
+        m.managedBy === "admin"
+          ? permissionsUtil.canCreateOfficialResources(m)
+          : permissionsUtil.canCreateMetric(m);
```

Comment:
We shouldn't block duplicating.  It's going to be fairly common for someone without admin permission to duplicate an official metric and tweak it, which is totally fine.  We should just make sure in that case, that when they duplicate, the "Official" status is not copied over if they don't have permission for it.